sale_project_fixed_price_task_completed_invoicing/README.md

```diff
@@ -0,0 +1,56 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+Sale Project Fixed Price Task Completed Invoicing
+=================================================
+
+The main goal of this module is to add the possibility to link a sale.order.line
+to a project.task considering the delivery.
+Unless sale_timesheet, the quantity shipped won't be linked to a Timesheet nor
+to the time spent on the task. The price is fixed on the sale.order.line and it
+will be considered as shipped once the task is accomplished.
+
+Usage
+=====
+
+Create a product with product.type 'Service' and track_service 'Completed Task'.
+
+Use it in a Sale Order. Once you validate the Sale Order, it will create a linked
+project and task.
+
+Once the task is finished, on the form view of the corrisponding task click on the
+button 'Invoiceable'. The linked sale.order.line will be considered as shipped.
+
```

Comment:
> Once the task is finished

How do you plan to set the task finished? Or it will be just the button "Invoiceable" ?